website/thaliawebsite/settings/production.py

```diff
@@ -35,9 +35,9 @@
     "default": {
         "ENGINE": "django.db.backends.postgresql",
         "USER": os.environ.get("POSTGRES_USER", "postgres"),
-        "PASSWORD": os.environ.get("POSTGRES_PASSWORD", ""),
+        "PASSWORD": os.environ.get("POSTGRES_PASSWORD", None),
```

Comment:
What is the difference between `""` and `None`? Does `""` mean an empty password and `None` mean no password is send at all?

Reply:
This is indeed the case